tony-history-server/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'play'
+    id 'idea'
+}
+
+def playVersion = "2.6.20"
+def hadoopVersion = "3.1.1"
+def scalaVersion = System.getProperty("scala.binary.version", "2.12")
```

Comment:
Please define these versions in the root build.gradle file and reuse them here. We shouldn't define hadoopVersion in 2 places.

You should be able to make a variable globally accessible by defining it like so:

```
ext.hadoopVersion = "3.1.1"
...
```

Then you should be able to access it from subprojects using `ext.hadoopVersion`